### PR TITLE
OSDOCS-2142: Release note for Azure subnet space usage in the machine…

### PR DESCRIPTION
**Pull request by mjpytlak**
https://issues.redhat.com/browse/OSDOCS-2142

Updated `Installation and upgrade` to include `Increased size of Azure subnets within the machine CIDR`

https://deploy-preview-35568--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes.html#ocp-4.9-azure-cidr

### CODE REVIEW

**Comment by mjpytlak** [comment]
@tsze-redhat This is the release note covering the Azure CIDR work. Please review. Thank you.

**Comment by tsze-redhat** [comment]
The language is same as corresponding change for GCP.
https://github.com/openshift/openshift-docs/pull/32054

LGTM since it is good for Matthew / GCP.

File: release_notes/ocp-4-9-release-notes.adoc [hunk 1]
```diff
@@ -58,6 +58,10 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-9-installation-and-upgrade"]
 === Installation and upgrade
 
+[id=ocp-4.9-azure-cidr]
+==== Increased size of Azure subnets within the machine CIDR
+The {product-title} installation program for Microsoft Azure now creates subnets as large as possible within the machine CIDR. This lets the cluster use a machine CIDR appropriately sized to accommodate the number of nodes in the cluster.
```
Comment (mburke5678) [inline release_notes/ocp-4-9-release-notes.adoc:63]: s/appropriately/that is appropriately ??
Might make the sentence a little more clear. I stumbled over that phrase the first time.

**Comment by mburke5678** [comment]
@mjpytlak One nit. Otherwise LGTM

**Comment by mjpytlak** [comment]
@mburke5678 Nice catch. Updated.

**Pull request merged**